orchestration/src/main/java/com/sap/ai/sdk/orchestration/OrchestrationConfigReference.java

```diff
@@ -0,0 +1,92 @@
+package com.sap.ai.sdk.orchestration;
+
+import com.google.common.annotations.Beta;
+import javax.annotation.Nonnull;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.RequiredArgsConstructor;
+import lombok.Value;
+
+/**
+ * Class representing a reference to an Orchestration config stored in prompt registry.
+ *
+ * @since 1.14.0
+ */
+@Value
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@Beta
+public class OrchestrationConfigReference {
```

Comment:
(Minor)

We could reduce future maintenance if the getters were package scoped(?)